sse2neon.h

```diff
@@ -111,19 +116,64 @@
 
 /* If using MSVC */
 #ifdef _MSC_VER
+#include <windows.h>
```

Comment:
If/when to include windows.h is a decision best left to the application. Including it here creates headaches for the project that was my whole motivation for looking at sse2neon in the first place ([ares-emulator/ares](https://github.com/ares-emulator/ares)). It avoids including windows.h in most translation units because doing so pollutes the global namespace with names that clash with application code.

The macros used in _mm_prefetch() could easily be replaced with constants. FlushInstructionCache() is the real problem case. I see three options:

1. Reproduce the FlushInstructionCache() declaration here. Ugly, a bit evil, but unlikely to cause issues in practice.
2. Guard this behind something the application must #define to opt in. I suspect most users would never even notice the absence of _mm_clflush().
3. Leave it like this and force users to deal with the side effects of including windows.h.

Just speaking personally, 3 is my least favorite option.

Reply:
Option 1 feels a bit rough to me - duplicating Win API functions feels icky. Option 3 I can see your point, so I suppose this leaves us with option 2 - will implement next commit

Reply:
Option 2 implemented - does that look acceptable?

Reply:
Fine by me!